### ISSUE TITLE
Feature proposal: show database constraints

### ISSUE THREAD

**Issue by ngr-t** [eralchemy/eralchemy#30]
Hi maintainers,

It looks that currently an output of eralchemy doesn't have database constraints such as `NOT NULL` or `UNIQUE`(I tried drawing from SQLAlchemy Base). I think it's helpful if they are drawn in the output.
How do you think of?

Anyway, thank you for creating this package.

**Comment by Alexis-benoist** [comment]
It's a good idea with a flag to activate it.

**Comment by evilaliv3** [comment]
Is this the reason why currently every foreign key is marked (0, 1) and never (1, 1)=

**Comment by evilaliv3** [comment]
[schema.pdf](https://github.com/Alexis-benoist/eralchemy/files/1644011/schema.pdf)

**Comment by psychemedia** [comment]
In `sqla.py` it looks like the relationship types are baked in from foreign key relationships:

```
def relation_to_intermediary(fk):
    """Transform an SQLAlchemy ForeignKey object to it's intermediary representation. """
    return Relation(
        right_col=format_name(fk.parent.table.fullname),
        left_col=format_name(fk._column_tokens[1]),
        right_cardinality='?',
        left_cardinality='*',
    )
```

I don't see anywhere it's possible to __set__ the full range of types `*: {0..N}`, `?: {0,1}`, `+: {1,N}`, `1: {1}`

Presumably, if we have a primary key defined we can assume it's `UNIQUE` (and `NOT NULL`?) in absence of those being declared explicitly?

**Comment by maurerle** [comment]
This is fixed in b6782f6e60412b26a3bc6abb091777fa02d85560 which fixed the behavior to also uses the primary keys unique constraint.
If there is anything still wrong, just open a new issue :)
